src/utils/utils.tsx

```diff
@@ -25,20 +27,36 @@ export const getGeography = (city: string[] | null | undefined, state: string[]
     return Translate("Not Reported");
   }
 
-  function renderOutGeography(geo: string[] | null | undefined) {
+  function renderOutGeography(geo: string[] | string | null | undefined) {
     if (!geo || geo.length === 0) {
       return undefined
     }
-    else if (geo.length > 1 && typeof geo !== "string") {
-      return geo?.join(", ");
+    if (typeof geo === "string"){
+      return geo.trim() + ", "
+    }
+    else if (geo.length > 1) {
+
+      return geo.join(", ") + ", ";
     }
-    return geo + ", "
+    return geo[0].trim() + ", "

```

Comment:
Don't think we need this line, the else if block above should work even if array is of length 1